### DISCUSSION TITLE
Problems with references in react.

### DISCUSSION THREAD

**Discussion by luas10c** [vercel/next.js#53956] [App Router]
I created a project with npx create-next-app example

I just made this code
![image](https://github.com/vercel/next.js/assets/66514698/ba4fde18-c891-4184-92fb-374e70392788)

I'm having these problems
![image](https://github.com/vercel/next.js/assets/66514698/643e51b0-d6d5-439d-9dca-2cd1a1e77962)

**Comment by luas10c** [comment]
I checked that it happens in the canary version

**Reply by Nadeera3784** [reply]
Hi @luas10c ,
Can you try with this? 
` console.log(ref.current) `

**Reply by luas10c** [reply]
I have the same problem, I understood that the error only appears when I use console.log

**Reply by luas10c** [reply]
if you use console.info the error does not happen.

**Reply by Nadeera3784** [reply]
My guess is some JavaScript in your code that overwrites console.log for compatibility reasons (this will also affect the console of the application's tab). You can verify that by pressing F12 in this tab and checking that console.log still works fine.

for example: 
![image](https://github.com/vercel/next.js/assets/22231476/eb97c973-7948-4f35-ae88-41a954703741)

**Reply by luas10c** [reply]
try giving console.log in a ref of an element as I showed the error in some cases, for example using "Children" from react.

**Reply by luas10c** [reply]
```tsx

'use client'

import { Children } from 'react'

export const Card = (props: Props) => {
  const { children } = props
  
  const items = Children.toArray(children)
  console.log(items)
  
  return (
    <div>
      <span>.card</span>
    </div>
  )
}

```


this code is to remove this error.

**Reply by luas10c** [reply]
I tested it on a Windows computer and the error doesn't happen.
